2021-06-18 04:00:40,203 INFO    MainThread:4030201 [internal.py:wandb_internal():91] W&B internal server running at pid: 4030201, started at: 2021-06-18 04:00:40.202640
2021-06-18 04:00:40,212 INFO    WriterThread:4030201 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_040038-2u4ftbg6/run-2u4ftbg6.wandb
2021-06-18 04:00:40,213 DEBUG   HandlerThread:4030201 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 04:00:42,406 DEBUG   HandlerThread:4030201 [meta.py:__init__():39] meta init
2021-06-18 04:00:42,406 DEBUG   HandlerThread:4030201 [meta.py:__init__():53] meta init done
2021-06-18 04:00:42,407 DEBUG   HandlerThread:4030201 [meta.py:probe():210] probe
2021-06-18 04:00:42,413 DEBUG   HandlerThread:4030201 [git.py:repo():33] git repository is invalid
2021-06-18 04:00:42,413 DEBUG   HandlerThread:4030201 [meta.py:_save_pip():57] save pip
2021-06-18 04:00:42,415 DEBUG   HandlerThread:4030201 [meta.py:_save_pip():71] save pip done
2021-06-18 04:00:42,415 DEBUG   HandlerThread:4030201 [meta.py:_save_conda():78] save conda
2021-06-18 04:00:46,478 DEBUG   HandlerThread:4030201 [meta.py:_save_conda():86] save conda done
2021-06-18 04:00:46,479 DEBUG   HandlerThread:4030201 [meta.py:probe():252] probe done
